app/Http/Controllers/EpisodeManager.php

```diff
@@ -23,9 +25,39 @@ class EpisodeManager extends Controller
     */
     public function index()
     {
-        $episodes = Episode::all();
 
-        return view('dashboard/pages/view_episodes', compact('episodes'));
+        $user                           = User::all();
+        $online_user                    = $user->where('active', 1)->count();
+        $offline_user                   = $user->where('active', 0)->count();
+        $numbers_of_users_on_suyabay    = $user->count();
+
+        $episodes                       = Episode::get();
+        $active_episode                 = $episodes->where('status', 1);
+        $pendding_episode               = $episodes->where('status', 0);
+
+        $channels                       = Channel::get();
+
+        $data = 
+        [
+            "user" => 
+                    [
+                        "total"     => $numbers_of_users_on_suyabay,
+                        "online"    => $online_user,
+                        "offline"   => $offline_user
+                    ],
+            
+            "episodes" =>
+                    [
+                        "recent"    => $episodes,
+                        "active"    => $active_episode,
+                        "pending"   => $pendding_episode
+                    ],
+
+            "channels" => $channels
+
+        ];
+
+        return view('dashboard/pages/index', compact('data'));
```

Comment:
**Code Quality - Nitpick**

Consistency is key. Other methods written by @andela-fokosun in this class used the dot syntax for the views.

Please use that. So it will something like 'dashboard.pages.index'. Please fix